python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -140,7 +140,7 @@ def fit(self, data, epochs=1, batch_size=32):
             if isinstance(data, tuple):
                 data = DataLoader(TensorDataset(torch.from_numpy(data[0]),
                                                 torch.from_numpy(data[1])),
-                                  batch_size=batch_size,
+                                  batch_size=batch_size//self.num_processes,
```

Comment:
We should report a warning if `batch_size` can not be divided with no remainder by `self.num_processes`

Reply:
OK

Reply:
just realized this should be `batch_size=max(1, batch_size//self.num_processes)`